android/app/src/main/python/mmrelay/plugin_loader.py

```
@@ -0,0 +1,938 @@
+                        try:
+                            # Check if we're running in a pipx environment
+                            in_pipx = (
+                                "PIPX_HOME" in os.environ
+                                or "PIPX_LOCAL_VENVS" in os.environ
+                            )
+
+                            if in_pipx:
+                                logger.info(
+                                    f"Attempting to install missing dependency with pipx inject: {missing_module}"
+                                )
+                                subprocess.check_call(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)